connectors/src/admin/cli.ts

```diff
@@ -383,6 +391,61 @@ const slack = async (command: string, args: parseArgs.ParsedArgs) => {
 
       break;
     }
+
+    case "clean-nango-connection-id": {
+      const slackConfigurations = await SlackConfiguration.findAll();
+      const connections = await nango_client().listConnections();
+      const slackConnections = connections.connections.filter(
+        (connection: {
```

Comment:
We also need to filter on the creation date, we can't revoke token that are younger than I would say one hour (to be very safe).
We could revoke a token that has just been created on Nango but not yet in connectors if we don't do that. (on top of the discussed edge case of replication lag latency).

Reply:
Done